src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$marginBotElement: 0 16px 16px;
+$height: 60px;
```

Comment:
Only two variables? Try to add all the main values (colors, card width, gaps etc)